spec/abci/abci++_basic_concepts.md

```diff
@@ -78,8 +78,8 @@ call sequences of these methods.
   proposer to perform application-dependent work in a block before proposing it.
   This enables, for instance, batch optimizations to a block, which has been empirically
   demonstrated to be a key component for improved performance. Method `PrepareProposal` is called
-  every time CometBFT is about to broadcast a Proposal message, but no previous proposal has
-  been locked at consensus level. CometBFT gathers outstanding transactions from the
+  every time CometBFT is about to broadcast a Proposal message and CometBFT's _validValue_ is `nil`. 
+  CometBFT gathers outstanding transactions from the
```

Comment:
I new line/paragraph here would be helpful.

Reply:
Fixed in the new version.